### DISCUSSION TITLE
How can I receive loading event from player in custom theme?

### DISCUSSION THREAD

**Discussion by leelai** [jhomlala/betterplayer#572] [Q&A] [answered]
Hi, I am trying to build custom ui but how can I receive loading event from controller? I know there is default ui for redefine theme, but if I want to use custom theme, how do I receive this event?

**Comment by jhomlala** [comment, marked as answer]
Better Player doesn't have "Loading event". In default themes (Material/Cupertino) widget detects that video is loading by checking `initialized` event which can be received from `BetterPlayerContoller`.
